dm/dm/master/scheduler/scheduler.go

```diff
@@ -394,7 +393,6 @@ func (s *Scheduler) addSource(cfg *config.SourceConfig) error {
 }
 
 // UpdateSourceCfg update the upstream source config to the cluster.
-// please verify the config before call this.
```

Comment:
some fields like `enable-gtid` should notice syncer. Please check other fields

Reply:
now refresh source config when task resume https://github.com/pingcap/tiflow/pull/4760/commits/3965843ef74e47cbc5c6681c4bb60177464bade5

Reply:
@lance6716 ptal again 🙇‍♂️🙇‍♂️🙇‍♂️